### DISCUSSION TITLE
Is it possible to draw dotted line?

### DISCUSSION THREAD

**Discussion by ruimo** [emilk/egui#1607] [Q&A] [answered]
Hi, I want to draw a rectangle with dotted line to select more than one shapes at once. The egui::Stroke seems to have only color and line width. Any ideas?

**Comment by emilk** [comment, marked as answer]
You can use https://docs.rs/epaint/latest/epaint/enum.Shape.html#method.dotted_line

**Reply by ruimo** [reply]
Thanks! The dotted_line() seems draw nothing (may be my usage is wrong..) but dashed_line() worked.
